### ISSUE TITLE
"grpc_channel_arguments" flag does not have effect in 1.7 version

### ISSUE THREAD

**Issue by cosnina** [tensorflow/serving#1019]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Darwin
- **TensorFlow Serving installed from (source or binary)**: source
- **TensorFlow Serving version**: 1.7

### Describe the problem
When I run TF Serving by setting the flag 
*--grpc_channel_arguments=grpc.max_connection_age_ms=2000*

The TF Serving logs show this error during the prediction:
```
grpc.max_connection_age_ms ignored: it must be an integer
```
Same problem when I run the unit tests (https://github.com/tensorflow/serving/blob/r1.7/tensorflow_serving/model_servers/tensorflow_model_server_test.py).

### Exact Steps to Reproduce
1. Run TF Serving 1.7 by passing --grpc_channel_arguments flag. 
E.g.:
--grpc_channel_arguments=grpc.max_connection_age_ms=2000

2. Run a client (e.g. python client) for prediction

### Source code / logs
The error is here: https://github.com/tensorflow/serving/blob/r1.7/tensorflow_serving/model_servers/main.cc

line 309. The fix should be:
```
if (tensorflow::strings::safe_strto32(channel_argument.value, &value)) {
```

Affected unit test: https://github.com/tensorflow/serving/blob/r1.7/tensorflow_serving/model_servers/tensorflow_model_server_test.py

**Comment by chrisolston** [comment]
Thanks for explaining the bug in detail. If you can prepare a PR with the fix that would be great!

**Comment by netfs** [comment]
does this happen on master branch or latest 1.10.0 release? if yes, lets have
the fix in (and sorry for the delay in getting the PR in).

else i'd not bother honestly. 1.7 is kinda old and this does not seem to be a
critical fix, that deserves a cherrypick on 1.7 release (and a rebuild+release
of new patch level).

**Comment by netfs** [comment]
This is fixed upstream via #871 (and in latest TF release too).
We do not intend to backport/cherrypick the fix to (now very old) 1.7 release.

Thanks!
